src/ServiceInsight/Shell/LicenseRegistrationViewModel.cs

```diff
@@ -123,7 +121,7 @@ public void Extend()
 
         public void ContactSales()
         {
-            network.Browse("http://particular.net/extend-your-trial-45");
+            network.Browse("mailto:contact@particular.net?subject=ServiceInsight");
```

Comment:
What happens if no mail client is installed?

Reply:
https://stackoverflow.com/questions/27176849/what-does-mailto-do-when-there-is-no-email-client

TL;DR: I don't think it's something we should concern ourselves with.

Reply:
When balancing effort vs. value we decided to go with mailto instead of a webform on the website, so this is an acceptable risk (although Windows and Mac both come with mail clients preinstalled)